apps/aecore/test/aec_eunit_SUITE.erl

```diff
@@ -130,10 +130,13 @@ application_test(Config) ->
     meck:expect(aec_fork_block_settings, lima_extra_accounts, 0, []),
     meck:expect(aec_fork_block_settings, lima_contracts, 0, []),
     meck:expect(aec_fork_block_settings, block_whitelist, 0, #{}),
+    meck:new(aeu_info, []),
+    meck:expect(aeu_info, block_info, 0, 591),
```

Comment:
Will this need to be updated every release?

Reply:
no :)